rfd/0097-teleport-connect-usage-metrics.md

```diff
@@ -0,0 +1,175 @@
+---
+authors: Grzegorz Zdunek (grzegorz.zdunek@goteleport.com)
+state: draft
+ ---
+
+# RFD 97 - Teleport Connect usage metrics
+
+## Required Approvers
+
+* Engineering:? @ravicious @zmb3
+* Product: @klizhentas @xinding33
+
+## What
+
+Collect downloads and usage metrics of Teleport Connect.
+
+## Why
+
+Currently, the team has no information on how many users download, install and use Teleport Connect on a daily basis.
+In order to effectively plan the development of the product, the team should also know what is the adoption of new
+features, which ones are the most popular and which are problematic for users.
+
+## Details
+
+### Collecting events
+
+Events for Connect will be collected on the client side. TypeScript code will have a stateless metrics service that will
+forward them to the gRPC handler exposed by `tsh daemon`. To prevent flooding backend with a large number of small
```

Comment:
```suggestion
forward them to the gRPC handler exposed by `tsh daemon`, which will ultimately submit them to a service called `prehog`. To prevent flooding backend with a large number of small
```

I'd mention prehog before you refer to it.